src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -56,210 +155,194 @@ public new IEnumerable ItemsSource
-                return _itemsTemplate;
+                return itemsTemplate;
             }
             set
             {
-                _itemsTemplate = value;
+                itemsTemplate = value;
                 base.ItemsTemplate = value;
                 if (value == null)
                 {
                     //layouter.clear()
                     return;
                 }
 
-                NeedInitalizeLayouter = true;
+                needInitalizeLayouter = true;
 
-                if (_itemsLayouter != null && InternalItemSource != null)
+                if (itemsLayouter != null && InternalItemSource != null)
                 {
-                    _itemsLayouter.Initialize(this);
-                    _itemsLayouter.RequestLayout(0.0f, true);
+                    itemsLayouter.Initialize(this);
+                    itemsLayouter.RequestLayout(0.0f, true);
                     if (ScrollingDirection == Direction.Horizontal)
                     {
                         ContentContainer.SizeWidth =
-                                _itemsLayouter.CalculateLayoutOrientationSize();
+                                itemsLayouter.CalculateLayoutOrientationSize();
                     }
                     else
                     {
                         ContentContainer.SizeHeight =
-                            _itemsLayouter.CalculateLayoutOrientationSize();
+                            itemsLayouter.CalculateLayoutOrientationSize();
                     }
-                    NeedInitalizeLayouter = false;
+                    needInitalizeLayouter = false;
                 }
             }
-        }
+        }        
 
-        private ItemsLayouter _itemsLayouter = null;
         /// <summary>
         /// Items Layouter.
         /// </summary>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
         [EditorBrowsable(EditorBrowsableState.Never)]
         public new ItemsLayouter ItemsLayouter
         {
             get
             {
-                return _itemsLayouter;
+                return itemsLayouter;
             }
             set
             {
-                _itemsLayouter = value;
+                itemsLayouter = value;
                 base.ItemsLayouter = value;
                 if (value == null)
                 {
-                    NeedInitalizeLayouter = false;
+                    needInitalizeLayouter = false;
                     return;
                 }
 
-                if ((_itemsSource != null) && (_itemsTemplate != null))
+                if ((itemsSource != null) && (itemsTemplate != null))
                 {
                     if (InternalItemSource == null)
                     {
-                        InternalItemSource = ItemsSourceFactory.Create(this, _itemsLayouter);
+                        InternalItemSource = ItemsSourceFactory.Create(this, itemsLayouter);
```

Comment:
ItemsSourceFactory.Create's first parameter is IEnumerable itemsSource.
Is it correct if "this"(CollectionView) is sent to ItemsSourceFactory.Create's first argument?

Reply:
This `Create()` will call 
```cs
IGroupableItemSource Create(CollectionView colView, ItemsLayouter layouter)
```